lib/resources/orders.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @name Orders
+ * @description This module exposes functions
+ *              related to the `/orders` path.
+ *
+ * @module Orders
+ **/
+
+import { curry } from 'ramda'
+
+import routes from '../routes'
+import request from '../request'
+
+const findAll = curry((opts, pagination) =>
```

Comment:
I know that this `pagination` variable looks like a pattern because it shows up in  a lot of routes, but looks like in this case which you are receiving the whole request body and this does more things than just paging.
```suggestion
const findAll = curry((opts, body) =>
  request.get(opts, routes.orders.base, body || {})
```